src/hotspot/cpu/s390/stubRoutines_s390.cpp

```diff
@@ -55,8 +55,8 @@ void StubRoutines::zarch::generate_load_absolute_address(MacroAssembler* masm, R
     __ load_const_optimized(Z_R0, table_addr);
     __ z_cgr(table, Z_R0);  // safety net
     __ z_bre(L);
-    __ z_illtrap();
     __ asm_assert(Assembler::bcondEqual, "crc_table: external word relocation required for load_absolute_address", 0x33);
```

Comment:
How should this `asm_assert` ever get hit? It's preceded by a `bre.`

Reply:
This was requested change (initially it was not part of this PR).

Reply:
The request was to "move z_illtrap() after asm_assert()". For debug builds, you then get a descriptive assert message. For release builds, you at least have a security net.

Reply:
Seems like you didn't get my point. The `z_bre` is fine, but the assert condition "bcondEqual" can't be true after that.

Reply:
Arrgh! You are right. Using the inverse condition (bcondNotEqual) is a better choice.

Reply:
> What about using `stop` or anything which doesn't take a condition? That would avoid confusion.

That is a good idea. It's of no benefit to check for the complement of the condition above.

Reply:
Thanks! That's much better readable. Note that there are more places like https://github.com/openjdk/jdk/blob/5635d301d89f62874969e475b53dcbd155aa56cb/src/hotspot/cpu/s390/stubRoutines_s390.cpp#L68. In addition, you don't need the illtrap after stop any more. Would be great if you could clean it up.

Reply:
done..